core/src/main/java/tc/oc/pgm/api/Config.java

```diff
@@ -81,6 +81,13 @@
    */
   Duration getCycleTime();
 
+  /**
+   * Gets a duration to countdown before restarting the server.
```

Comment:
```suggestion
   * Gets a duration to wait before restarting the server.
```
I think this would align with the other duration descriptions better.

Reply:
Had it like this originally but it could be muddled with the idle time regardless will switch it back.